src/cic/io/read.clj

```diff
@@ -108,6 +108,8 @@
                              (update :param parse-double)))
                    (map (juxt :name :param))
                    (into {}))]
+    (when (empty? coefs)
+      (throw (Exception. "Tried to load empty joiner csv")))
```

Comment:
I'd prefer this to be `(ex-info "Tried to load empty joiner csv" {<map with some useful information like the params to the function>})`